geometry_msgs/Pose[] poses
float32[] scores
geometry_msgs/Point[] samples
int32[] object_ids
